.github/workflows/deploy.yml

```diff
@@ -9,11 +9,11 @@ on:
         type: choice
         default: review
         options:
-        - qa
-        - review
-        - staging
-        - sandbox
-        - dv
+          - qa
+          - review
+          - staging
+          - sandbox
+          - dv
```

Comment:
This seems like an odd change.

Reply:
My IDE did it automatically when I updated that file 🤷🏻 presumably it thought the formatting was wrong.

I can revert this particular chunk if needed. But VS Code will do it again next time I edit the file 🫠